code/modules/admin/topic.dm

```diff
@@ -1759,8 +1759,10 @@
 			usr << "The person you are trying to contact is not wearing a headset"

```

Comment:
@KorPhaeron So, you want to add periods where it is needed in these sentences? Okay.